tests/fv3_conf/control_run.IN

```diff
@@ -166,6 +166,18 @@ if [ $IMP_PHYSICS = 8 ]; then
   cp    @[INPUTDATA_ROOT]/FV3_fix/CCN_ACTIVATE.BIN .
 fi
 
+#TEMPO
+if [ $IMP_PHYSICS = 88 ]; then
+  cp    @[INPUTDATA_ROOT]/FV3_fix/CCN_ACTIVATE.BIN .
+  rm qr_acr_qsV2.dat
+  rm qr_acr_qgV2.dat
+  rm freezeH2O.dat
+  cp /scratch2/BMC/wrfruc/jensen/ufs_tables/MP_TEMPO_freezeH2O.dat .
```

Comment:
@grantfirl @dustinswales should we create a new input directory like @[INPUTDATA_ROOT]/ufs_tables? or @[INPUTDATA_ROOT]/FV3_fix/ufs_tables?

Reply:
I think these files should be added to a new directory, @[INPUTDATA_ROOT]/FV3_input_data_TEMPO, just like the other scheme specific input data.
@AndersJensen-NOAA

Reply:
@dustinswales Could you modify the description to denote that TEMPO requires modified input data?

Reply:
@dustinswales I created and rsynced @[INPUTDATA_ROOT]/FV3_input_data_TEMPO on hera and derecho. You can go ahead to update. I will go ahead to test on derecho first. and I will rsync for the rest of the machines as well.